packages/compass-sidebar/src/components/csfle-connection-modal/csfle-connection-modal.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  css, spacing, Banner, Description, Label, Link, Toggle, Modal
+} from '@mongodb-js/compass-components';
+
+const toggleStyles = css({
+  marginBottom: spacing[3],
+  marginRight: spacing[3]
+});
+
+function CSFLEConnectionModal({ csfleMode, open, setOpen, setConnectionIsCSFLEEnabled }) {
+  return (<Modal
+    open={open}
+    title="Client-Side Field-Level Encryption"
+    trackingId="csfle_connection_modal"
+    setOpen={setOpen}
+  >
+    <div>
+      <p>
+          This connection is configured with Client-Side Field-Level Encryption enabled.
+      </p>
+      <Toggle
```

Comment:
To get the toggle label and the toggle vertically aligned I'm thinking it might be help to wrap the `Toggle` and `Label` in a `div` with some styles to align them. How do you feel about:
```ts
const toggleContainerStyles = css({
  display: 'flex',
  alignItems: 'center'
});
```

```suggestion
      <div className={toggleContainerStyles}>
        <Toggle
```

It would look something like
<img width="636" alt="Screen Shot 2022-04-12 at 1 49 35 PM" src="https://user-images.githubusercontent.com/1791149/163023644-f277a243-f1fa-40e2-8057-7cdfa7c67eb3.png">

Reply:
Perfect, thank you!